docs/metadata.md

```diff
@@ -22,378 +22,206 @@ searching for particular kinds of data.
 
-## Transport Block
-
-This section describes the on the wire transport protocol, normally HTTP, but with scope to describe out-of-band 
-transports with an initial HTTP negotiation process. It contains at least a single `http` key, the value of which
-must be valid [Open|API|](https://swagger.io/specification/) 
-
-For example:
-
-```yaml
-transport:
-  http:
-    # This block is mandatory, and contains the Open|API| spec for the secured or open
-    # HTTP endpoints (depending on data class)
-    openapi: 3.0.0
-    info:
-      title: Sample |API|
-      description: CSV format data
-      version: 0.1.0
-    servers:
-      - url: http://data-provider-example.com
-        description: Describe this particular server if needed
-    paths:
-      "/data":
-        get:
-          summary: Returns a CSV containing all the data
-          description: If we had any more to describe, we'd do it here
-          responses:
-            '200':
-              description: CSV data stream
-```
+`dcat:endpointURL`
+: The URL of this specific instance of the API. It is interpolated into the `url` specified in the OpenAPI file using the `endpointURL` variable.
 
-**NOTE**: Because [API](glossary.md#term-Application-programming-interface) security is defined in relation to the data sensitivity class of the data set, it is not necessary to 
-define the security of any presented [API](glossary.md#term-Application-programming-interface) in this section. Data sets in class [IB1OE-O](glossary.md#term-Data-sensitivity-class-open) must expose an [API](glossary.md#term-Application-programming-interface) with no extra
-security measures, and those in [IB1OE-SA](glossary.md#term-Data-sensitivity-class-shared-A) and [IB1OE-SB](glossary.md#term-Data-sensitivity-class-shared-B) must be secured by [FAPI](glossary.md#term-Financial-Grade-API) using the Ib1 Trust FrameworkOpen Energy trust services.
+`ib1:heartbeatDescription`
+: The URL of an OpenAPI file (with Server specified as `dcat:endpointDescription`), which contains a single Path with a 200 response defined. This term will typically be the URL of one of a small number of standard OpenAPI files published in the Registry.
 
-### Heartbeat URL
+Any additional metadata defined by published Standards may be added.
 
-Data providers **SHOULD** create a secured endpoint to act as a heartbeat - if this is specifed then the [TFOEGS](glossary.md#term-Trust-FrameworkOpen-Energy-Governance-Service) will 
-periodically call it to assertain liveness and optionally gather metrics as described in
-[Heartbeat and monitoring endpoint](ops_guidelines/data_provider_ops_guidelines.md#heartbeat-and-monitoring-endpoint)
+## Dataset metadata fields
```

Comment:
Add the data assurance metadata fields - example usage:
```
    "ib1:trustFramework": "http://wstf.registry.ib1.org",
    "ib1:sensitivityClass": "IB1-O",
    "ib1:datasetAssurance": "IcebreakerOne.DatasetLevel1",
```
See https://icebreakerone.org/sops/assured-data-publishing/ for descriptions

Reply:
I've added the trust framework and assurance terms, although I wonder whether the values of latter need to be documented more?
Also, should both sensitivity and assurance level be URLs?

Reply:
You're right, [they should](https://jazz.net/wiki/bin/view/LinkedData/UseUrisForEnums). Without a fallback this would be a breaking change for SSE, who are currently using the non-namespaced versions. 

Let's make a new card for this change, and we can raise it with SSE first so we co-ordinate with them. It will also require an update to the Assurance webpage and definitions. This is fine because we're overdue to publish a revised edition anyway.